### PR TITLE
Docs: Add help file how to compile Bitcoin by one console command

### PR DESCRIPTION
**Pull request by 3s3s**
Simpliest way to compile Bitcoin

### CODE REVIEW

File: doc/build-onecommand.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+### Instruction
+This console command will build last (0.16.2) version Bitcoin clients for Linux 64 bit, Windows 64 bit and Windows 32 bit. 
+
+Has been tested on the clean server with Ubuntu 16 (64 bit 4GB RAM)
```
Comment (MarcoFalke) [inline doc/build-onecommand.md]: Note that building on Xenial is no longer supported

File: doc/build-onecommand.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
...dev bsdmainutils libboost-system-dev libboost-filesystem-dev libboost-chrono-dev libboost-program-options-dev libboost-test-dev libboost-thread-dev -y && apt-get update -y && add-apt-repository ppa:bitcoin/bitcoin && apt-get update -y && apt-get install libdb4.8-dev libdb4.8++-dev libminiupnpc-dev libzmq3-dev libqt5gui5 libqt5core5a libqt5dbus5 qttools5-dev qttools5-dev-tools libprotobuf-dev protobuf-compiler libqrencode-dev g++-mingw-w64-i686 mingw-w64-i686-dev g++-mingw-w64-x86-64 mingw-w64-x86-64-dev curl -y && update-alternatives --config x86_64-w64-mingw32-g++ &&  wget https://github.com/bitcoin/bitcoin/archive/v0.16.2.tar.gz && tar -xvzf v0.16.2.tar.gz && cd bitcoin-0.16.2/depends && make download && make && cd .. && ./autogen.sh && ./configure --enable-glibc-back-compat --prefix=`pwd`/depends/x86_64-pc-linux-gnu LDFLAGS="-static-libstdc++" && make clean && make && make install && cd depends && make HOST=x86_64-w64-mingw32 -j4 && cd .. && ./configure --prefix=`pwd`/depends/x86_64-w64-mingw32 && make clean && make && make install && cd depends && make HOST=i686-w64-mingw32 -j4 && cd .. && ./configure --prefix=`pwd`/depends/i686-w64-mingw32 && make clean && make && make install
```
Comment (MarcoFalke) [inline doc/build-onecommand.md]: Note that you shouldn't build and run untrusted input (the tar.gz's)

**Comment by practicalswift** [comment]
I’m afraid this documentation will get stale quickly as the normal build documentation is updated. Also I’m not conviced that building in one long command is something we want to encourage since that will obscure exactly where in the process any build failure occurs which will lead to bad bug reports.

Please consider looking at the existing build documentation instead and see if you can make it better.

**Comment by laanwj** [comment]
I don't really see a point of adding multiple parallel ways to build, sorry. Maintaining one per platform is enough.
The drawback of putting it all in one command is that it 's harder to read, review, and to maintain.

It could make sense to host this on your own site or blog, but NACK on merging.

> Also I’m not conviced that building in one long command is something we want to encourage since that will obscure exactly where in the process any build failure occurs which will lead to confused users and bad bug reports.

Exactly.

File: doc/build-onecommand.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+### Instruction
+This console command will build last (0.16.2) version Bitcoin clients for Linux 64 bit, Windows 64 bit and Windows 32 bit.
```
Comment (luke-jr) [inline doc/build-onecommand.md]: If it's going in the repo, please don't put the version here.

**Pull request closed without merging**